core/README.md

```diff
@@ -42,7 +42,7 @@ add `cloudevents-json-jackson` as a dependency and then:
 ```java
 import io.cloudevents.core.format.EventFormatProvider;
 
-EventFormat format = EventFormatProvider.getInstance().resolveFormat("application/json");
+EventFormat format = EventFormatProvider.getInstance().resolveFormat("application/cloudevents+json");
```

Comment:
Can you use this constant? https://github.com/cloudevents/sdk-java/blob/12cca34d213be499464edebcc012efaafe069bfa/formats/json-jackson/src/main/java/io/cloudevents/jackson/JsonFormat.java#L31